src/ToWords.ts

```diff
@@ -79,9 +93,19 @@ export class ToWords {
     return this.locale;
   }
 
-  public convert(number: number, options: ConverterOptions = {}): string {
+  /**
+   *
+   * @param number - user input number to be converted
+   * @param options - configuration set by user
+   * @returns {string} - converted number to words string
+   */
+  public convert(number: number | string, options: ConverterOptions = {}): string {
     options = Object.assign({}, this.options.converterOptions, options);
 
+    //check type of the user input and replace any commas
+    // and convert the input to number if string
+    number = this.checkType(number);
```

Comment:
I'd prefer the method to be named `clean` rather than `checkType`.

```ts
number = this.clean(number);
```
 is more readable.

Reply:
Updated the function name change and comments